### ISSUE TITLE
The 2.4.2.4 and 2.4.2.3.20150913 tags are not on the master or maintenance branches

### ISSUE THREAD

**Issue by asr** [agda/agda#1687]
Nils wrote:

> It seems as if the 2.4.2.4 tag is not on the master or maintenance branches.
> 
> If I start from the "top-most" commit on one of these branches, and follow DAG edges backwards, then I don't find this tag. In other words, the tag is not present in the output of

```
git log --graph --oneline --decorate --branches=master --branches=maint-2.4.2
```

> however, it is present in the output of

```
git log --graph --oneline --decorate --tags --branches=master --branches=maint-2.4.2
```

**Comment by asr** [comment]
Fixed.

``` bash
$ git log --graph --oneline --decorate --branches=maint-2.4.2 | grep 'tag: 2.4.2.4'
* 0b37390 (tag: 2.4.2.4) [ 2.4.2.4 release ] Removed Cabal test suite.
```

Since I removed the tag from upstream and pushed it again, I suggest (see note below) delete the "old" tag and fetch the "new" tag by doing:

``` bash
$ git tag -d 2.4.2.4
$ git fetch origin tag 2.4.2.4
```

Note: Since I didn't re-tag (see 'Discussion on Re-tagging' in Git [reference manual](https://git-scm.com/docs/git-tag)), I'm not sure if the above commands are strictly necessaries.

**Comment by asr** [comment]
_Correction_: Since I removed the tag from upstream and pushed it again, you should delete the old tag and fetch the new tag by doing:

``` bash
$ git tag -d 2.4.2.4
$ git fetch origin tag 2.4.2.4
```

You can test which tag you have by doing

``` bash
$ git rev-parse 2.4.2.4
```

which should return

```
4dfb0a57e2b43948c02455915b66b9dac29ac034
```

if you have the new version.

**Comment by asr** [comment]
I also fixed the issue for the 2.4.2.3.20150913 tag, so it is necessary to run

``` bash
$ git tag -d 2.4.2.3.20150913
$ git fetch origin tag 2.4.2.3.20150913
```

and `git rev-parse` should return

``` bash
$ git rev-parse 2.4.2.3.20150913
bb563d9ecfd68ee5fc8a828f8c81098009c719c7
```

**Comment by asr** [comment]
For the record, the 2.4.2.3.20150913 tag was fixed by

``` bash
maint $ git tag -d 2.4.2.3.20150913
maint $ git push origin :refs/tags/2.4.2.3.20150913
maint $ git tag -a 2.4.2.3.20150913 -m "Agda 2.4.2.4 release candidate" 6a4fd3ca8a4fba24ebb16a4617291ed192a792d3
maint $ git push --tags
```
